src/cmd/cli/command/compose.go

```diff
@@ -78,7 +78,7 @@ func makeComposeUpCmd() *cobra.Command {
 
 			var managedResources = GetUnreferencedManagedResources(project)
 			if len(managedResources) > 0 {
-				term.Warnf("Defang cannot monitor status of the following managed resources %v. Please use the AWS console directly.", managedResources)
+				term.Warnf("Defang cannot monitor status of the following managed service(s): %v.\n   - Please edit compose file, add to the \"depends_on\" section of any services requiring them.", managedResources)
```

Comment:
Updating text. Playground users do not have access to AWS console. Ask them to put managed services in their depends_on section so we can use logs to determine if their app is up.